src/features/opentracingZipkin-0.30.mf

```diff
@@ -1,7 +1,9 @@
 IBM-Feature-Version: 2
 IBM-ShortName: opentracingZipkin-0.30
 Subsystem-Content: com.ibm.websphere.appserver.opentracing-1.0; type="osgi.subsystem.feature",
- com.ibm.ws.opentracing.zipkin; version="[1.0.0,1.0.200)"
+ com.ibm.ws.opentracing.zipkin; version="[1.0.0,1.0.200)",
+ com.ibm.websphere.appserver.jaxrs-2.0; type="osgi.subsystem.feature"; ibm.tolerates:=2.1,
```

Comment:
Interesting that this zipkin sample didn't recognize CDI and JAX-RS as dependencies previously...  The changes look good for the Java EE 8 toleration.  I'm just questioning why these were not identified previously.

Reply:
It previously inherited them from opentracing-1.0 but with "tolerates" this feature explicitly needs to opt in and so needs to now declare then as dependencies.